### PR TITLE
[Doc] update desc for how to configure memory size for fe

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/faq/Others.md [hunk 1]
```diff
@@ -188,7 +188,14 @@ Run the `select current_version();` command or the CLI command `./bin/show_fe_ve
 
 ## How to set the memory size of an FE?
 
-FEs are used to store metadata. You can set the memory size of an FE based on the number of tablets. In StarRocks, you can set the memory size of an FE to 20 GB at most. 10 million tablets occupy about 20 GB of an FE memory.
+The metadata is stored in the memory used by the FE. You can set the memory size of the FE according to the number of tablets as shown in the diagram below. For example, if the number of tablets is below 1 million, you should allocate a minimum of 16GB of memory to the FE. You can configure Xms and Xmx values at the **JAVA_OPTS=** in the **fe.conf** file, and the Xms and Xmx values should be consistent. Additionally, the configuration should be same across all FEs because each FE can be a Leader.
+
+| Number of tablets    | Memory size of an FE |
```
Comment (amber-create) [inline docs/faq/Others.md:193]: image
-> Memory size per FE

File: docs/faq/Others.md [hunk 2]
```diff
@@ -188,7 +188,14 @@ Run the `select current_version();` command or the CLI command `./bin/show_fe_ve
...
+| -------------- | ----------- |
+| below 1 million      | 16 G        |
+| 1 ～ 2 million | 32 G        |
+| 2 ～ 5 million | 64 G        |
+| 5 ～ 10 million   | 128 G       |
```
Comment (amber-create) [inline docs/faq/Others.md:198]: image
G -> GB

File: docs/faq/Others.md [hunk 3]
```diff
@@ -188,7 +188,14 @@ Run the `select current_version();` command or the CLI command `./bin/show_fe_ve
 
 ## How to set the memory size of an FE?
 
-FEs are used to store metadata. You can set the memory size of an FE based on the number of tablets. In StarRocks, you can set the memory size of an FE to 20 GB at most. 10 million tablets occupy about 20 GB of an FE memory.
+The metadata is stored in the memory used by the FE. You can set the memory size of the FE according to the number of tablets as shown in the diagram below. For example, if the number of tablets is below 1 million, you should allocate a minimum of 16GB of memory to the FE. You can configure Xms and Xmx values at the **JAVA_OPTS=** in the **fe.conf** file, and the Xms and Xmx values should be consistent. Additionally, the configuration should be same across all FEs because each FE can be a Leader.
```
Comment (amber-create) [inline docs/faq/Others.md:191]: image
-> 16-GB memory

Comment (amber-create) [inline docs/faq/Others.md:191]: image
Add code format

Comment (amber-create) [inline docs/faq/Others.md:191]: image
-> any of the FEs can be elected as a Leader

File: docs/faq/Others.md [hunk 4]
```diff
@@ -188,7 +188,14 @@ Run the `select current_version();` command or the CLI command `./bin/show_fe_ve
 
 ## How to set the memory size of an FE?
 
-FEs are used to store metadata. You can set the memory size of an FE based on the number of tablets. In StarRocks, you can set the memory size of an FE to 20 GB at most. 10 million tablets occupy about 20 GB of an FE memory.
+The metadata is stored in the memory used by the FE. You can set the memory size of the FE according to the number of tablets as shown in the table below. For example, if the number of tablets is below 1 million, you should allocate a minimum of 16 GB memory to the FE. You can configure the values of the parameters `-Xms` 和 `-Xmx` in the **JAVA_OPTS** in the **fe.conf** file, and the values of the parameters `-Xms` 和 `-Xmx` should be consistent. Note that the configuration should be same across all FEs because any of the FEs can be elected as a Leader.
```
Comment (amber-create) [inline docs/faq/Others.md:191]: image
Add "configuration item" or like following "**JAVA_OPTS**"

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-2.5

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2

**Pull request merged**